### PR TITLE
A few useful edda links (AMI / Launch Configuration)

### PR DESCRIPTION
**Pull request by ajordens**

### CODE REVIEW

**Comment by ajordens** [comment]
Find these two particularly useful when debugging

File: app/scripts/modules/netflix/serverGroup/awsServerGroupDetails.html [hunk 1]
```diff
@@ -291,6 +291,16 @@ <h3 select-on-dbl-click>
             Cluster ({{ctrl.serverGroup.cluster}})
           </a>
         </li>
+        <li>
+          <a target="_blank" ng-href="http://edda-main.{{ctrl.serverGroup.region}}.{{ctrl.serverGroup.account}}.netflix.net:7001/REST/v2/aws/launchConfigurations/{{ctrl.serverGroup.launchConfig.launchConfigurationName}}">
```
Comment (ajordens) [inline app/scripts/modules/netflix/serverGroup/awsServerGroupDetails.html]: At some point we might/will get a complaint that the edda links don't work for anything but the prod/test accounts (edda-main vs. edda-mce).

File: app/scripts/modules/netflix/serverGroup/awsServerGroupDetails.html [hunk 2]
```diff
@@ -282,15 +282,25 @@ <h3 select-on-dbl-click>
     <collapsible-section heading="Edda Links">
       <ul>
         <li>
-          <a target="_blank" ng-href="http://edda-main.{{ctrl.serverGroup.region}}.{{ctrl.serverGroup.account}}.netflix.net:7001/REST/v2/aws/autoScalingGroups/{{ctrl.serverGroup.name}}">
+          <a target="_blank" ng-href="http://edda-{{ctrl.serverGroup.accountDetails.accountId}}.{{ctrl.serverGroup.region}}.{{ctrl.serverGroup.accountDetails.environment}}.netflix.net:7001/REST/v2/aws/autoScalingGroups/{{ctrl.serverGroup.name}}">
```
Comment (ajordens) [inline app/scripts/modules/netflix/serverGroup/awsServerGroupDetails.html]: This actually makes these links work in the different accounts.

Previously they were busted in mce (_for example_).

File: app/scripts/modules/amazon/serverGroup/details/serverGroupDetails.aws.controller.js [hunk 3]
```diff
@@ -370,5 +373,11 @@ module.exports = angular.module('spinnaker.serverGroup.details.aws.controller',
       }
       return null;
     };
+
+    this.getAccountDetails = (serverGroup) => {
```
Comment (anotherchrisberry) [inline app/scripts/modules/amazon/serverGroup/details/serverGroupDetails.aws.controller.js]: `getAccountDetails` implies that this method returns the account details. Maybe a better name would be `addAccountDetails` or `addAccountDetailsToServerGroup`.

**Comment by anotherchrisberry** [comment]
LGTM except for the name of the method.

**Pull request merged**